### PR TITLE
Use material design style in Supported Agencies View

### PR DESCRIPTION
**Pull request by cagryInside**
A material design list view was implemented for `supported agencies view`.
device-2016-04-18-115227
device-2016-04-18-115058

### CODE REVIEW

File: onebusaway-android/src/main/java/org/onebusaway/android/ui/MaterialListItem.java [hunk 1]
```diff
@@ -13,24 +13,31 @@
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
-package org.onebusaway.android.report.ui.model;
+package org.onebusaway.android.ui;
 
 /**
  * Model for report type
```
Comment (barbeau) [inline onebusaway-android/src/main/java/org/onebusaway/android/ui/MaterialListItem.java]: Could you please update the comment to reflect that this is a more generic list item with material design now?

**Comment by barbeau** [comment]
Thanks @cagryInside, this looks good!  One small comment inline.

File: onebusaway-android/src/main/java/org/onebusaway/android/ui/MaterialListItem.java (hunk 1 above)
Comment (cagryInside) [inline onebusaway-android/src/main/java/org/onebusaway/android/ui/MaterialListItem.java]: Fixed via https://github.com/OneBusAway/onebusaway-android/pull/464/commits/e54128318c2a48dfc8cab14fb78d071c4ef50230

**Comment by barbeau** [comment]
Thanks @cagryInside!

**Pull request merged**